tests/modules/keyspace_events.c

```diff
@@ -45,6 +45,8 @@ RedisModuleDict *loaded_event_log = NULL;
 /** stores all the keys on which we got 'module' keyspace notification **/
 RedisModuleDict *module_event_log = NULL;
 
+static size_t dels = 0;
```

Comment:
please add a comment describing this:
something like `counts how many deleted KSN we got on keys with a prefix of "count_dels_"`.